modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+Two annotations must be available to isolate a workload into partitioned resources. Annotate the containing namespace with `workload.openshift.io/allowed` and the pod
+with `target.workload.openshift.io`.
+
+Required OCP platform pods and namespaces are already annotated. To add extra pods to the partition, such as supported day 2 operators or any user-added catalog sources, annotate both the namespace and pod on creation.
```

Comment:
s/OCP platform/{product-title}

"day 2" is informal language.  "post-installation" might be an alternative.